pkg/nuctl/command/common/importreport.go

```diff
@@ -56,6 +57,13 @@ func (pr *ProjectReports) SprintfError() string {
 	return report
 }
 
+func (pr *ProjectReports) PrintAsTable(t table.Writer, onlyFailed bool) {
+	t.AppendHeader(table.Row{"type", "name", "status", "fail description", "auto-fixable"})
```

Comment:
Move this row to a const and reuse in both PrintAsTable
```
table.Row{"type", "name", "status", "fail description", "auto-fixable"}
```

Reply:
nice idea